src/ansible_navigator/image_manager/puller.py

```diff
@@ -142,18 +151,33 @@ def prologue_stdout(self):
         print("Updating the execution environment")
         print("\u002d" * width)
 
+    def _generate_pull_command(self) -> List:
+        """Generate the pull command.
+
+        :returns: The list of command parts
+        """
+        command_line = [self._container_engine, "pull"]
+        if isinstance(self._arguments, list):
```

Comment:
Would this `isinstance` check and the one above go away if we defaulted the new setting to `[]`?

Reply:
It would, but a default of `[]` gets really odd when you see the default in the `--help` because at the command line we use nargs + and append, and there's not really a way to express a list object at the CLI.